Added FAQ and troubleshooting section (#7124)
segmentio/segment-docs: src/connections/destinations/catalog/actions-amplitude/index.md

"traits" : {"$add": {"friendCount": 3} }
"traits" : {"$add": {"friendCount": 1} }
```
## FAQ and troubleshooting

> ### Why isn't the Session_Id being automatically added to my WEB events?

Reviewer: ```suggestion
### Why doesn't Segment automatically add the `session_id` to my web events?
```

> For the Session_Id to be automatically added to events, your browser must allow the following request URL to load:
> 
> ```https://cdn.segment.com/next-integrations/actions/amplitude-plugins/..```
> 
> To check if this request is being loaded, [inspect the network requests](https://developer.chrome.com/docs/devtools/network) on your website and look for 'Amplitude.' If the request is being blocked, please ensure it is allowed on your side.

Reviewer: ````suggestion
For Segment to automatically add the `session_id` to events, your browser must allow the following request URL to load:

```
https://cdn.segment.com/next-integrations/actions/amplitude-plugins/..
```

To check if you are loading this request, [inspect the network requests](https://developer.chrome.com/docs/devtools/network){:target="_blank”} on your website and look for 'Amplitude.' If the request is not loading, confirm it is allowed on your side.
````
